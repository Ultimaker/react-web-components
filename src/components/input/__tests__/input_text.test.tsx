import React from 'react';
import { shallow } from 'enzyme';
import * as Input from '..';

describe('input_text.tsx:', () => {
    test('snapshot', () => {
        expect(
            shallow(<Input.Text />),
        ).toMatchSnapshot();
    });
});
