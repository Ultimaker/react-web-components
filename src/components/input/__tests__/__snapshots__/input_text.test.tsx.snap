// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`input_text.tsx: snapshot 1`] = `
<input
  className="input-text"
  type="text"
/>
`;
